docs/topics/funding.md

```diff
@@ -115,36 +111,50 @@ REST framework continues to be open-source and permissively licensed, but we fir
 
 ---
 
-## Making the business case
+## What funding has enabled so far
 
-Our [successful Kickstarter campaign](https://www.kickstarter.com/projects/tomchristie/django-rest-framework-3) demonstrates the impressive cost-reward ratio of shared funding of open-source software.
+* The [3.4](http://www.django-rest-framework.org/topics/3.4-announcement/) and [3.5](http://www.django-rest-framework.org/topics/3.5-announcement/) releases, including schema generation for both Swagger and RAML, a Python client library, a Command Line client, and addressing of a large number of outstanding issues.
+* The [3.6](http://www.django-rest-framework.org/topics/3.6-announcement/) release, including  JavaScript client library, and API documentation, complete with auto-generated code samples.
+* Tom Christie, the creator of Django REST framework, working on the project full-time.
+* Over 320 closed issues on [GitHub](https://github.com/tomchristie/django-rest-framework) since Tom Christie started working on the project full-time (June 2016 - March 2017).
```

Comment:
Taking in issues *and* PRs I see ~850 since june 2016. - https://github.com/tomchristie/django-rest-framework/issues?utf8=%E2%9C%93&q=closed%3A%3E2016-06-01

Perhaps we could phrase it in a way that it'll be valid for longer. Eg. "Around 80-90 issues and pull requests closed per month", or something like that?